src/procedures/ForceTransfer.ts

```diff
@@ -0,0 +1,54 @@
+import { Procedure } from './Procedure';
+import { ProcedureTypes, PolyTransactionTags, ForceTransferArgs, ErrorCodes } from '../types';
+import { PolymathError } from '../PolymathError';
+import { isValidAddress } from '../utils';
+
+export class ForceTransfer extends Procedure<ForceTransferArgs> {
+  public type = ProcedureTypes.ForceTransfer;
+
+  public async prepareTransactions() {
+    const { symbol, value, from, to, log, data } = this.args;
+    const { securityTokenRegistry } = this.context;
+    const addresses: { [key: string]: string } = { from, to };
+
+    /**
+     * Validation
+     */
+
+    Object.keys(addresses).forEach(key => {
+      if (!isValidAddress(addresses[key])) {
+        throw new PolymathError({
+          code: ErrorCodes.InvalidAddress,
+          message: `Provided "${key}" address is invalid: ${addresses[key]}`,
+        });
+      }
+    });
+
+    const securityToken = await securityTokenRegistry.getSecurityToken({
+      ticker: symbol,
+    });
+
+    if (!securityToken) {
+      throw new PolymathError({
+        code: ErrorCodes.FetcherValidationError,
```

Comment:
This one should be `ProcedureValidationError`. I just noticed that the `ChangeDelegatePermission` and `EnableGeneralPermissionManager` procedures also have the wrong error code. Could you squeeze in that fix in this PR?